legal-api/requirements/bcregistry-libraries.txt

```diff
@@ -1 +1 @@
-git+https://github.com/bcgov/business-schemas.git@2.5.12#egg=registry_schemas
+git+https://github.com/bcgov/business-schemas.git#egg=registry_schemas
```

Comment:
Your requirements.txt will need the correct version.

Reply:
updated